src/components/Cat.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from "react"
+import {connect} from "react-redux"
+import {removeCat} from "./../action_creators"
+
+function Cat ({src, fact, id, dispatch}) { // todo fix lint: react/jsx-no-bind
+  return (<div className="cat" onClick={() => dispatch(removeCat(id))}>
```

Comment:
bind action creators is fine but honestly - in larger applications you would use your containers to maintain all state/dispatch actions. Ergo, removeCat should be  a function in App (doesn't make as much sense for this toy example, but would for larger apps - so is good practice) that you pass down as props. Keeps all state logic in one area (essentially your container components act as controllers, with all stateless function components being strictly view)